src/components/Accounts/ViewFeesFines/ViewFeesFines.js

```diff
@@ -332,43 +356,29 @@ class ViewFeesFines extends React.Component {
 
     return (
       <Dropdown
-        onSelectItem={this.handleOptionsChange}
+        renderTrigger={this.renderToggle}
+        usePortal
       >
-        <Button data-test-ellipsis-button data-role="toggle" buttonStyle="hover dropdownActive">
-          <strong>•••</strong>
-        </Button>
-        <DropdownMenu id="ellipsis-drop-down" data-role="menu">
-          <MenuItem itemMeta={{ a, action: 'pay' }}>
-            <Button disabled={!((elipsis.pay === false) && (buttonDisabled === false))} buttonStyle="dropdownItem">
-              <FormattedMessage id="ui-users.accounts.history.button.pay" />
-            </Button>
-          </MenuItem>
-          <MenuItem itemMeta={{ a, action: 'waive' }}>
-            <Button disabled={!((elipsis.waive === false) && (buttonDisabled === false))} buttonStyle="dropdownItem">
-              <FormattedMessage id="ui-users.accounts.history.button.waive" />
-            </Button>
-          </MenuItem>
-          <MenuItem itemMeta={{ a, action: 'refund' }}>
-            <Button disabled={!((elipsis.refund === false) && (buttonDisabled === false))} buttonStyle="dropdownItem">
-              <FormattedMessage id="ui-users.accounts.history.button.refund" />
-            </Button>
-          </MenuItem>
-          <MenuItem itemMeta={{ a, action: 'transfer' }}>
-            <Button disabled={!((elipsis.transfer === false) && (buttonDisabled === false))} buttonStyle="dropdownItem">
-              <FormattedMessage id="ui-users.accounts.history.button.transfer" />
-            </Button>
-          </MenuItem>
-          <MenuItem itemMeta={{ a, action: 'cancel' }}>
-            <Button disabled={!((elipsis.error === false) && (buttonDisabled === false))} buttonStyle="dropdownItem">
-              <FormattedMessage id="ui-users.accounts.button.error" />
-            </Button>
-          </MenuItem>
+        <DropdownMenu id="ellipsis-drop-down">
+          <this.MenuButton disabled={!((ellipsis.pay === false) && (buttonDisabled === false))} account={a} action="pay">
```

Comment:
These `disabled` checks could be rewritten as, e.g., `disabled={ellipsis.pay || buttonDisabled}` for improved breviclarity.

Reply:
Breviclaritynessified!